src/docs/product/sentry-basics/index.mdx

```diff
@@ -1,10 +1,89 @@
 ---
-title: Sentry Basics
+title: Get Started with Sentry
 sidebar_order: 10
-description: "Welcome to Sentry Basics, our primer on using tools available in sentry.io to help you resolve issues quickly."
+description: "These tutorials will get you started with using Sentry."
 redirect_from:
   - /basics/
   - /product/sentry-basics/guides/
 ---
 
+## What is Sentry?
+
+Sentry is a software monitoring tool that helps developers identify and fix code-related issues. From error tracking to performance monitoring, Sentry provides code-level observability that makes it easy to diagnose issues and learn continuously about your application code health.
```

Comment:
```suggestion
## What is Sentry?

Sentry is a software monitoring tool that helps developers identify and fix code-related issues. From error tracking to performance monitoring, Sentry provides code-level observability that makes it easy to diagnose issues and learn continuously about your application code health.

## How to Get the Most Out of Sentry

There are three things we recommend you do to get the most out of Sentry:

- **Connect your codebase**, being sure you've also set up [source code management integrations](/product/integrations/source-code-mgmt/) and [source maps](/product/sentry-basics/integrate-frontend/upload-source-maps/) (if applicable). This will allow Sentry to link errors and performance issues directly to the problematic line of code via readable stack traces and suspect commits.
- **Set up [alerts](/product/alerts/)** to make sure you know when things you care about are going wrong. Be sure to follow [alert best practices](/product/alerts/best-practices/), so that you only get notified about things you care about.
- **Customize [Discover](/product/discover-queries/), [Dashboards](/product/dashboards/), and [queries](/product/discover-queries/query-builder/)** to unlock custom insights about the things that are going wrong in your app. 
```

I think this is a REALLY important section and should live at the top.